pkg/version/version.go

```diff
@@ -92,5 +91,42 @@ func GetLatestVersion(uuid string, source string) (string, error) {
 		return "", err
 	}
 
-	return versionRsp["version"], nil
+	channel := parseChannel(Version)
+	if channel == "" {
+		channel = "version"
```

Comment:
if our local `Version` does not include a channel, should we consider it malformed and return an error?